sklearn/gaussian_process/tests/test_kernels.py

```diff
@@ -185,6 +185,7 @@ def test_kernel_stationary():
 
 def test_kernel_clone():
     """ Test that sklearn's clone works correctly on kernels. """
+    bounds = (1e-5, 1e5)
     for kernel in kernels:
         kernel_cloned = clone(kernel)
 
```

Comment:
The test below looks at `__dict__` which is now not useful. You'll need to go looking for `__dict__` magic in tests.